policyengine_us/parameters/gov/states/nc/tax/use_tax/cap.yaml

```diff
@@ -0,0 +1,136 @@
+description:  North Carolina limits the use tax to this cap, based on adjusted gross income.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  label: North Carolina use tax cap
+  reference:
+    - title: NCDOR Use Tax
+      href: https://www.ncdor.gov/taxes-forms/sales-and-use-tax/use-tax
+    - title: 2022 North Carolina Form D-401, Line 18
+      href: https://www.ncdor.gov/2022-d-401-individual-income-tax-instructions/open#page=23
+brackets:
+  - threshold:
+      2022-01-01: 0
+    amount:
+      2022-01-01: 1
+  - threshold:
+      2022-01-01: 2_200
+    amount:
+      2022-01-01: 2
+  - threshold:
+      2022-01-01: 3_700
+    amount:
+      2022-01-01: 3
+  - threshold:
+      2022-01-01: 5_200
+    amount:
+      2022-01-01: 4
+  - threshold:
+      2022-01-01: 6_700
+    amount:
+      2022-01-01: 5
+  - threshold:
+      2022-01-01: 8_100
+    amount:
+      2022-01-01: 6
+  - threshold:
+      2022-01-01: 9_600
+    amount:
+      2022-01-01: 7
+  - threshold:
+      2022-01-01: 11_100
+    amount:
+      2022-01-01: 8
+  - threshold:
+      2022-01-01: 12_600
+    amount:
+      2022-01-01: 9
+  - threshold:
+      2022-01-01: 14_100
+    amount:
+      2022-01-01: 10
+  - threshold:
+      2022-01-01: 15_600  
+    amount:
+      2022-01-01: 11
+  - threshold:
+      2022-01-01: 17_000
+    amount:
+      2022-01-01: 12
+  - threshold:
+      2022-01-01: 18_500
+    amount:
+      2022-01-01: 13
+  - threshold:
+      2022-01-01: 20_000
+    amount:
+      2022-01-01: 14
+  - threshold:
+      2022-01-01: 21_500
```

Comment:
add 2021 values

Reply:
2022 and 2023 values were same as 2021, so i just changed the year to be 2021